docs/client_reference.rst

```diff
@@ -255,6 +255,9 @@ The client session supports the context manager protocol for self closing.
                    (optional). *json* and *data* parameters could not
                    be used at the same time.
 
+      :param dict cookies: HTTP Cookies to send with
+                           the request (optional)
```

Comment:
1. `.. versionadded:: 3.5` tag is required,
2. Please add a sentence about merging global session cookies with explicitly passed ones.

Reply:
OK, docs added